### PR TITLE
Note about fork deprecation

### PR DESCRIPTION
**Pull request by mlewand**
We're not using this repository for a long time. It's worth to put a note so that people are not confused with it.

We add a notice that in 3 months it will be removed.

On a PR side of things I added just a super short message, decorated with fancy emojis.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 🚨 We Have Moved! 🚨
```
Comment (wojtekidd) [inline README.md]: Please use [Sentence case for titles](https://github.com/cksource/new-websites/issues/101) so _We have moved!_

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+This repository is **no longer active**, we've moved development to [ckeditor/ckeditor-dev](https://github.com/ckeditor/ckeditor-dev) repository.
```
Comment (wojtekidd) [inline README.md]: We're missing something before _development_ so _CKEditor 4 development_ would be nice.

Comment (AnnaTomanek) [reply README.md]: This repository is **no longer active** as we moved CKEditor 4 development to the [ckeditor/ckeditor-dev](https://github.com/ckeditor/ckeditor-dev) repository.

File: README.md (hunk 1 above)
Comment (AnnaTomanek) [reply README.md]: Actually the rest of the document uses Title Case, let's leave it. It's deprecated anyway.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+
+We're going to remove this deprecated repository in December 2017.
```
Comment (AnnaTomanek) [inline README.md]: We are going...

**Pull request merged**